Sources/Basic/OutputByteStream.swift

```diff
@@ -125,48 +159,50 @@ public class OutputByteStream: OutputStream {
     /// does not write any other characters (like the quotes that would surround
     /// a JSON string).
     public func writeJSONEscaped(_ string: String) {
+        var utf8 = [UInt8]()
```

Comment:
Why add a temporary here? This should be able to use the single-byte write function and expect it to be reasonably efficient.